src/batches/setBatch3Dto4D.m

```diff
@@ -0,0 +1,34 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function matlabbatch = setBatch3Dto4D(volumesList, outputName, dataType, RT)
```

Comment:
Wanna try writing a test for that one? :wink:

Reply:
I had a look at the other tests but I am not sure I am able to understand how to do it

Reply:
Hum... Indeed... This one might need to create some dummy data.

I could adapt the `createDummyData.sh` script to do that.